### PR TITLE
Rename app in pom.xml

### LINKED ISSUE #62: Bug: pom.xml file lists the project name as "example"
Currently, the pom.xml file lists the maven project name as "example". This does not affect the user experience but is a simple fix.

### PR DESCRIPTION
**Pull request by iain801**
## Overview

In this PR, the name and description of the app are changed in the pom.xml file. 

## Linked Issues

Closes #62

### CODE REVIEW

**Comment by publicqi** [comment]
No files changed?

**Review by MPriston — DISMISSED** [review]
Change is pretty strait forward, changes don't break code, LGTM

File: pom.xml [hunk 1]
```diff
@@ -11,10 +11,10 @@
         <relativePath /> <!-- lookup parent from repository -->
     </parent>
     <groupId>edu.ucsb.cs156</groupId>
-    <artifactId>example</artifactId>
+    <artifactId>happiercows</artifactId>
```
Comment (dscpsyl) [inline pom.xml]: The name should be happycows and not happiercows

File: pom.xml [hunk 2]
```diff
@@ -11,10 +11,10 @@
...
     <version>1.1.0</version>
-    <name>example</name>
-    <description>Spring Boot + React Example</description>
+    <name>happiercows</name>
```
Comment (dscpsyl) [inline pom.xml]: Also here too

**Review by dscpsyl — CHANGES_REQUESTED** [review]

File: pom.xml (hunk 2 above)
Comment (iain801) [reply pom.xml]: have we got approval for this from the professor?

**Review by yuanchenmeng — APPROVED** [review]
Current code LGTM!

**Review by dscpsyl — APPROVED** [review]

**Review by pconrad — APPROVED** [review]
LGTM!

**Pull request merged**